pkg/web/templates/apidocs.html

```diff
@@ -20,6 +20,7 @@
     allow-spec-file-load = "false"
     allow-spec-file-download = "true"
     schema-style = "table"
+    schema-description-expanded	= "true"
```

Comment:
Some of the schema descriptions are long. This change would ensure that the full description is displayed by default instead of requiring users to click `Multiline description` for more details.